src/io_uring.h

```diff
@@ -0,0 +1,21 @@
+#ifndef IO_URING_H
+#define IO_URING_H
+
+#include "server.h"
+
+/* Initialize io_uring at server startup if have io_uring configured, setup io_uring submission and completion. */
+void initIOUring(void);
+
+/* If the client is suitable to use io_uring handle the write request. */
+int writeUsingIOUring(client *c);
```

Comment:
This name says "write" but this function doesn't write.

It is used as a boolean, so name it something boolean like `isIoUringEnabled` or `canWriteUsingIoUring`.

Reply:
I like `canWriteUsingIOUring`.